src/chrome/content/rules/Matcc.company.xml

```diff
@@ -0,0 +1,14 @@
+<!--
+	Server error:
+		- matcc.company, equivalent to www.matcc.company
```

Comment:
This should not be redirected to `www` to be consistent with the http variant.

Reply:
Fixed.